qemu/tests/cfg/cpu_device_hotplug_maximum.cfg

```diff
@@ -47,7 +48,10 @@
             vcpu_cores = 0
             vcpu_threads = 8
     variants:
+        no Host_RHEL.m8.u4
+        required_qemu = [6.0.0-16, )
```

Comment:
Is this work? They are under `variants` but without any specific variant.

Reply:
> Is this work? They are under `variants` but without any specific variant.

Thanks and changed.